examples/quickstart_snowflake/quickstart_snowflake/__init__.py

```diff
@@ -1 +1,35 @@
-from .repository import quickstart_snowflake
+from dagster_snowflake import build_snowflake_io_manager
+from dagster_snowflake_pandas import SnowflakePandasTypeHandler
+
+from dagster import (
+    Definitions,
+    ScheduleDefinition,
+    define_asset_job,
+    load_assets_from_package_module,
+)
+
+from . import assets
+
+daily_refresh_schedule = ScheduleDefinition(
+    job=define_asset_job(name="all_assets_job"), cron_schedule="0 0 * * *"
+)
+
+
+defs = Definitions(
+    assets=load_assets_from_package_module(assets),
+    resources={
+        "io_manager": build_snowflake_io_manager([SnowflakePandasTypeHandler()]).configured(
```

Comment:
switch to `snowflake_pandas_io_manager`

Reply:
good catch! this was built right before that was released so i totally forgot to switch

Reply:
updated